.github/workflows/_android-build-test.yml

```diff
@@ -71,10 +71,9 @@ jobs:
 
       - name: Calculate docker image
         id: calculate-docker-image
-        uses: ./.github/actions/calculate-docker-image
+        uses: huydhn/test-infra/.github/actions/calculate-docker-image@calculate-docker-image
```

Comment:
They will be replaced with main branch after https://github.com/pytorch/test-infra/pull/4397 is reviewed and landed